api/envoy/config/core/v3/extension.proto

```diff
@@ -28,3 +30,32 @@ message TypedExtensionConfig {
   // <config_overview_extension_configuration>` for further details.
   google.protobuf.Any typed_config = 2 [(validate.rules).any = {required: true}];
 }
+
+// Configuration source specifier for a late-bound extension configuration. The
+// parent resource is warmed until all the initial extension configurations are
+// received, unless the flag to apply the default configuration is set.
+// Subsequent extension updates are atomic on a per-worker basis, and apply to
+// new streams while the active streams continue using the older extension
```

Comment:
nit: same comment about streams. Check all of this to make sure it's generic.

Reply:
Updated.